backend/app.js

```diff
@@ -74,7 +75,7 @@ app.use(function(req, res, next) {
   next();
 });
 
-var port = process.env.PORT || 3000;
+var port = process.env.PORT || 3005;
```

Comment:
setup script was made for windows machine and port, I will update port #, will revert setup script as well.